documentation/site/pages/zkc/staking.mdx

```diff
@@ -0,0 +1,47 @@
+# Staking
+
+Token holders can stake $ZKC to earn passive staking rewards, as well as participate in governance. 
+
+## Staking $ZKC using the Boundless CLI
+
+TODO
+
+## Staking Rewards
+
+Staking rewards are earned by token holders who have staked $ZKC. These rewards are distributed every reward epoch. $ZKC staking rewards are emitted every reward epoch. Each epoch lasts 2 days, resulting in 182.5 epochs per year.
+
+### Emission Rate
+
+The initial supply is 1 billion ZKC. Emissions are divided equally across epochs throughout the year. The annual emission rate decreases over time based on the years since token launch:
+
+* Year 0: ~7.0% annual  
+* Year 1: ~6.5% annual  
+* Year 2: ~6.0% annual  
+* Year 3: ~5.5% annual  
+* Year 4: ~5.0% annual  
+* Year 5: ~4.5% annual  
+* Year 6: ~4.0% annual  
+* Year 7: ~3.5% annual  
+* Year 8+: ~3.0% annual (minimum rate)
+
+### Emissions Distribution
+
+Emissions occur at the end of each epoch. Every epoch's emissions are allocated to two external contracts for distribution:
+
+#### Passive Pool
+
+25% of epoch emissions are divided proportionately to stakers. For example, if you stake 10% of the $ZKC in the passive staking pool, you will receive 2.5% of the total emissions i.e. 10% of 25%.
+
+#### Active Pool
+
+- A maximum of 75% of epoch emissions are given to provers who have generated proofs during the epoch. 
+- Provers receive a reward for proving based on the percentage of proving (this is tracked via the number of cycles) they have carried out during the epoch. 
+- Provers must stake $ZKC to be eligible for these rewards. Each prover has a reward cap based on their staked $ZKC. This reward cap is set to `staked amount / 15`. 
+    - Therefore if a prover has staked 150 $ZKC, they are eligible to receive a maximum of 10 $ZKC every epoch. If a prover would have received more $ZKC due to their cycle contribution but under-staked $ZKC, the delta remains unclaimed which can be distributed at a later date.
```

Comment:
the delta remains unclaimed for the epoch, and the DAO can choose to distribute it at a later date.